app/src/main/java/org/simple/clinic/drugs/search/DrugsSearchScreen.kt

```diff
@@ -49,6 +52,11 @@ class DrugsSearchScreen : BaseScreen<
   @Inject
   lateinit var router: Router
 
+  @Inject
+  lateinit var drugFrequencyFactory: DrugFrequencyFactory
+
+  private var drugFrequencyToFrequencyChoiceItemMap: Map<DrugFrequency?, DrugFrequencyChoiceItem>? = null
```

Comment:
Is there a reason why it has to be a mutable property? and also can't we just have this inside the `onViewCreated` as well?